octobot_tentacles_manager/cli.py

```diff
@@ -167,6 +172,8 @@ def register_tentacles_manager_arguments(tentacles_parser) -> None:
     tentacles_parser.add_argument("-c", "--creator", help="Start OctoBot Tentacles Creator.\n Examples: -c Evaluator "
                                                           "to create a new evaluator tentacles. Use: -c help to get the"
                                                           " Tentacle Creator help.", nargs='+')
+    tentacles_parser.add_argument("-cy", "--cythonized", help="Option for the --pack command: cythonize and "
```

Comment:
I don't understand why it's `cythonized` and not `cythonize` because the tentacle isn't already cythonized

Reply:
To get it in a `cythonized` way, if you think `cythonize` is better, we go for it, I have no strong opinion about this.

Reply:
I prefer `cythonize` :wink: